experimental/AlgebraicStatistics/src/serialization.jl

```diff
@@ -0,0 +1,58 @@
+@register_serialization_type PhylogeneticModel uses_params 
+@register_serialization_type GroupBasedPhylogeneticModel uses_params 
```

Comment:
if you set `use_params` the save mechanices will expect there to be a save_type_params function implemented.